test/data/EnumerableSet.ts

```diff
@@ -0,0 +1,195 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { bnToBytes32 } from '@solidstate/library';
+import {
+  EnumerableSetBytes32Mock,
+  EnumerableSetBytes32Mock__factory,
+} from '@solidstate/typechain-types';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+describe('EnumerableSet', async () => {
+  describe('Bytes32Set', async () => {
+    let instance: EnumerableSetBytes32Mock;
+    let deployer: SignerWithAddress;
+
+    beforeEach(async () => {
+      [deployer] = await ethers.getSigners();
+      instance = await new EnumerableSetBytes32Mock__factory(deployer).deploy();
+    });
+
+    describe('__internal', () => {
+      const zeroBytes32 = bnToBytes32(ethers.constants.Zero);
```

Comment:
There's also `ethers.constants.HashZero`, but your way is probably more clear.

Reply:
Awesome, I left it as is for consistency but I will be sure to keep this in mind when I need a byte32 sized zero hash.